src/Fields.php

```diff
@@ -428,23 +428,21 @@ protected function wf_crm_get_fields($var = 'fields') {
       $fields['activity_activity_date_time'] = array(
         'name' => t('Activity # Date'),
         'type' => 'date',
-        'value' => 'now',
+        'default_value' => 'now',
       );
       $fields['activity_activity_date_time_timepart'] = array(
         'name' => t('Activity # Time'),
-        'type' => 'webform_time',
-        'value' => 'now',
+        'type' => 'time',
+        'default_value' => 'now',
       );
```

Comment:
Critical bits are 'default_value' is the proper key and type should be 'time' -> not 'webform_time'. When it's 'time' -> the webform civicrm will find it and glue it to the date to create a proper activity date time stamp.

Reply:
Actually I was wrong on that!  Reverted back to webform_time.